api/v1/controllers/subjects.js

```diff
@@ -60,6 +60,26 @@ function checkDuplicates(tagsArr) {
 }
 
 /**
+ * Validates the correct filter parameter
+ * passed in query parameters
+ * @param {Array} filterParams Filter Tags Array
+ */
+function validateFilterParams(filterParams) {
+  let subjectTagsCounter = 0;
```

Comment:
Instead of this...
```
+  let subjectTagsCounter = 0;
+
+  for (let i = 0; i < filterParams.length; i++) {
+    if (filterParams[i][ZERO] === '-') {
+      subjectTagsCounter++;
+    }
+  }
```

... why not this... ?
```
const EXCLUDE_SYMBOL = '-';
subjectTagsCounter = filterParams
  .filter((i) => i.startsWith(EXCLUDE_SYMBOL)).length;
```